pages/docs/community/styleguide/inclusive-language.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Inclusive Language style guide
+description: This style guide outlines the guideline for using inclusive languages in documentation.
+---
+
+# Inclusive Language
+
+While writing documentation we are indirectly reaching out to thousands of people all around the globe. 
+Thus, we must ensure that our diverse audience can connect with our information. 
+
+### 1. Culturally Inclusive Language
+
+Some phrases and words are popularly used only in specific regions of the world, thus our audience from other parts 
+of the world might need help to relate to the same. Hence, we should try to avoid using such phrases in our documentation. 
+Some examples of Region Specific Language are -
+- “It’s not rocket science” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+- Phrases written in regional languages instead of English.
+- “It’s not a piece of cake” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+
+### 2. Gender Neutral Language - 
+
+Some phrases and words are targeting men or men/women specifically. This might result in other gender groups feeling 
+left out while reading our documentation. 
+Some examples of Gender Neutral Language are -
+- Using “they/them” instead of “his/him” or “she/her”.
+- Using “Hello everyone” instead of “Hello guys”.
+- Using “Chairperson” instead of “Chairman” or “Chairwoman”.
+
+### 3. People First Language - 
+
```

Comment:
Here are further examples of **People First Language** that you can add:
- Instead of saying "the disabled person," say "person with a disability."
- Instead of saying "the mentally ill," say "people with mental health conditions."
- Instead of saying "wheelchair-bound," say "wheelchair user" 
- Instead of saying "the homeless," say "people experiencing homelessness."

Remember that People First Language puts the person before their condition or characteristic while recognizing that **some minority groups actually take pride in seeing their disability as part of their cultural identity.** 

(personal example: I am autistic and I do NOT appreciate the term _"person with autism"_ because I am not ashamed of having a neurodiverse brain. I am proud to identify and be an **autistic woman**.)